File: src/dependent/fulfillment/fulfillment_result.py
class FulfillmentResult:
    def __init__(self, reservation_id: str, tracking_id: str) -> None:
        self._reservation_id = reservation_id
        self._tracking_id = tracking_id

    @property
    def reservation_id(self) -> str:
        return self._reservation_id

    @property
    def tracking_id(self) -> str:
        return self._tracking_id

File: src/dependent/fulfillment/FulfillmentResult.java
package org.example.dependent.fulfillment;

public class FulfillmentResult {
    private final String reservationId;
    private final String trackingId;

    public FulfillmentResult(String reservationId, String trackingId) {
        this.reservationId = reservationId;
        this.trackingId = trackingId;
    }

    public String getReservationId() {
        return reservationId;
    }

    public String getTrackingId() {
        return trackingId;
    }
}


